Soccar [Game-Server]/src/nl/soccar/gameserver/controller/socnet/message/EventMessage.java

```diff
@@ -0,0 +1,38 @@
+package nl.soccar.gameserver.controller.socnet.message;
+
+import nl.soccar.gameserver.controller.socnet.message.handler.EventMessageHandler;
+import nl.soccar.library.enumeration.EventType;
+import nl.soccar.socnet.message.Message;
+import nl.soccar.socnet.message.MessageConstants;
+import nl.soccar.socnet.message.MessageEvent;
+
+/**
+ * Message for changing host.
+ *
+ * @author PTS34A
+ */
+@MessageEvent(id = MessageConstants.EVENT_MESSAGE, handler = EventMessageHandler.class)
+public final class EventMessage extends Message {
+
+    private final EventType type;
+    private final int playerId;
+
+    public EventMessage(EventType type, int playerId) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)